File: QTD2.Infrastructure/Model/CSE_ILACertLink_PartialCredit/CSE_ILACertPartialCreditCreateUpdateOption.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace QTD2.Infrastructure.Model.CSE_ILACertLink_PartialCredit
{
    public class CSE_ILACertPartialCreditCreateUpdateOption
    {
        public List<CSE_ILACertPartialCredit> CSE_ILACertPartialCredits = new List<CSE_ILACertPartialCredit>();
    }

    public class CSE_ILACertPartialCredit
    {
        public int ClassScheduleEmployeeId { get; set; }
        public double? PartialCreditHours { get; set; }
        public List<CSE_ILACertSubRequirementPartialCredit> subRequirements = new List<CSE_ILACertSubRequirementPartialCredit>();
    }

    public class CSE_ILACertSubRequirementPartialCredit
    {
        public string Reqname { get; set; }
        public double? PartialCreditHours { get; set; }
    }
}

File: QTD.SPA/src/app/_DtoModels/CSE_ILACertLink_PartialCredit/CSE_ILACertPartialCreditCreateUpdateOption.ts
export class CSE_ILACertPartialCreditCreateUpdateOption{
    cSE_ILACertPartialCredits:CSE_ILACertPartialCredit[] = [];
}

export class CSE_ILACertPartialCredit{
    classScheduleEmployeeId:string;
    partialCreditHours?:number;
    subRequirements:CSE_ILACertSubRequirementPartialCredit[] = [];
}

export class CSE_ILACertSubRequirementPartialCredit{
    reqName:string;
    partialCreditHours?:number;
}